### PR TITLE
Add Pixel-Based Visual Information Fidelity measure

### PR DESCRIPTION
**Pull request by antoinedelplace**
Add an implementation of the Pixel-Based Visual Information Fidelity (PB-VIF) measure in the metrics folder. The new folder called "vif" is composed of:
- README.md: A documentation
- vif.py: the module where the pbvif function is located
- main.py: a driver to test the function
- lena_ref.bmp, lena_gauss_noise.bmp, lena_low_pass_filter.bmp, lena_jpeg.jpg: 4 images to validate the measure

The results are the same as the one provided by PyMetrikz but uses Tensorflow instead of Numpy.

### CODE REVIEW

File: metrics/vif/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Pixel-Based Visual Information Fidelity (PB-VIF)
```
Comment (shakes76) [inline metrics/vif/README.md]: Good

File: metrics/vif/main.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        name_ref = tab_image_filenames[0]
+        name_query = tab_image_filenames[1:]
+
+    img_ref = imageio.imread(name_ref, as_gray=True).astype(np.float32)
+    img_query = []
+    for i in range(0, len(name_query)):
+        img_query.append(imageio.imread(name_query[i], as_gray=True).astype(np.float32))
+    
+    vif_tab = vif.pbvif(img_ref, img_query)
+
```
Comment (shakes76) [inline metrics/vif/main.py]: Good to add inline comments even if original code didn't have them

**Comment by shakes76** [comment]
## TF Usage
Good
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback req
Request Description good

**Comment by zakajd** [comment]
If anyone is looking for Visual Information Fidelity metric on PyTorch and also googled this thread, decent implementation can be found in [ this repo ](https://github.com/photosynthesis-team/piq)
